components/BooleanCheckbox.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Checkboxes as LBHCheckboxes,
+  CheckboxItem
+} from "lbh-frontend-react/components";
+import React from "react";
+import {
+  DynamicComponentControlledProps,
+  DynamicComponent
+} from "remultiform/component-wrapper";
+
+import PropTypes from "../helpers/PropTypes";
+
+export type BooleanCheckboxProps = DynamicComponentControlledProps<string> & {
```

Comment:
```suggestion
export type BooleanCheckboxProps = DynamicComponentControlledProps<boolean> & {
```

This should be using a boolean value in the database. I think we want it to have a fixed value (e.g. `"enabled"`), and we check if the value is checked. This _might_ be easier if it didn't use `Checkboxes`.